### DISCUSSION TITLE
How can I change background and export wgt to use in my TV?

### DISCUSSION THREAD

**Discussion by doblejote** [jhassan8/crunchyroll-tizen#105] [Q&A]
Really thank you for projects like this that still work perfectly on my television model with Tizen 7. What I want to ask is if I can, for example, modify the background of the project so that it is my favorite anime and export it again as an application to install it, whichever I realize that from my Tizen Studio, when I export it again, it no longer lets me install it with a certificate error. My question is how can I get a valid certificate that allows me to install it on my television and modify it? Thank you

**Comment by jhassan8** [comment]
Hi @doblejote, You have the option of using the same certificate that the project has, so you have to do it this way.

1. Set tizen var
    - Linux: ```export PATH=$PATH:<your_tizen_path>/tizen-studio/tools/ide/bin```
    - Windows: ```set PATH=%PATH%;<your_tizen_path>/tizen-studio/tools/ide/bin``` 
2. In project folder run:
    ```npm install```
    ```npm run tizen-build```
    ```npm run tizen-package```

This generates the .wgt, keep in mind that this will install the offline version with your change, if a new update appears in the app you will have to install the app again.

**Reply by doblejote** [reply]
Muchas gracias por la respuesta, eso era lo que me estaba ocurriendo que al modificar algo me sale que no es válido para instalar en mi TV.

Pruebo e informo sobre los resultados

**Reply by doblejote** [reply]
![image](https://github.com/jhassan8/crunchyroll-tizen/assets/91273079/04aaa2f9-0fa1-4c6c-927d-52915303ac20)

Está usando los certificados que encuentra en mi Tizen Studio, debo hacer algo adicional como usar el comando --sign
